Gems/PhysX/Code/Source/PhysXCharacters/Components/EditorCharacterControllerComponent.h

```diff
@@ -67,11 +70,8 @@ namespace PhysX
             required.push_back(AZ_CRC_CE("TransformService"));
         }
 
-        static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
+        static void GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
```

Comment:
Empty function can be removed again, I think

Reply:
Same answer as before.